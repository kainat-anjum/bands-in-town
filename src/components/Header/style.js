import styled from "styled-components";
import {ContainerPadding} from '../../commons/styles'
import {device,colors} from '../../commons/variables'

export const HeaderStyled = styled.div`
  ${ContainerPadding}
  
  display: flex;
  align-items: center;
  justify-content: space-between;
  background-color: ${colors.darkBlueColor};
  height: 100px;

  @media ${device.tablet} {
    flex-direction: column;
    align-items: flex-start;
    padding: 20px 50px;
  }
`;