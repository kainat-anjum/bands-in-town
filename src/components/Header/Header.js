import React from "react";

import { HeaderStyled } from "./style";

export const Header = (props) => {
  return <HeaderStyled>{props.children}</HeaderStyled>;
};
