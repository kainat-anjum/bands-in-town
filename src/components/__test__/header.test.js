import React from "react";
import { cleanup } from "@testing-library/react";
import renderer from "react-test-renderer";
import { Header } from "../Header/Header";

afterEach(cleanup);

describe("Testing Header Data", () => {
  it("Tests children", () => {
    const tree = renderer
      .create(
        <Header>
          <p>
            Trending Artists: Dark Alley | Dream Your Moments, Until I met you,
            Gimme some courage
          </p>
        </Header>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
