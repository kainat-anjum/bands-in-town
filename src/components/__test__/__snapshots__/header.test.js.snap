// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Header Data Tests children 1`] = `
<div
  className="sc-bdvvaa iVXsnw"
>
  <p>
    Trending Artists: Dark Alley | Dream Your Moments, Until I met you, Gimme some courage
  </p>
</div>
`;
